### DISCUSSION TITLE
WPF ListBox SelectedItems TwoWay Binding

### DISCUSSION THREAD

**Discussion by giscus** [Tyrrrz/Tyrrrzme#24] [Blog]
# WPF ListBox SelectedItems TwoWay Binding

For some unclear reasons, WPF's ListBox control does not allow two-way binding on the SelectedItems property the way it does with SelectedItem. This could have been very useful when using multi-select to bind the whole list of selected items to the view…

https://tyrrrz.me/blog/wpf-listbox-selecteditems-twoway-binding

**Comment by hjy1210** [comment]
Can you provide a simple full wpf app example?

**Reply by Tyrrrz** [reply]
I'm fortunately not using WPF anymore, so I can't

**Comment by hjy1210** [comment]
thanks for your reply.
